test/uk/gov/hmrc/gform/gform/StructuredFormDataBuilderSpec.scala

```diff
@@ -240,13 +250,48 @@ class StructuredFormDataBuilderSpec extends Spec {
         "decField" -> "decFieldValue"
       ),
       objectStructure(
-        "field"    -> textNode("fieldValue"),
-        "ackField" -> textNode("ackFieldValue"),
-        "decField" -> textNode("decFieldValue")
+        field("field", textNode("fieldValue"), Map.empty),
```

Comment:
Make our lives easy & remove the duplication by giving the alternateFieldNames parameter of field(...) a default, Map.empty.